bundles/net.tourbook.common/src/net/tourbook/common/weather/IWeather.java

```diff
@@ -212,6 +218,21 @@ public interface IWeather {
 
    };
 
+   /**
+    * Ids for the weather's air quality. Those Ids are saved in the db.
+    */
+   public static final String[] airQualityIds          = new String[] {
```

Comment:
Is SonarCloud not displaying a warning that the name is not UPPERCASE ?

Reply:
It would (rule [S116](https://sonarcloud.io/organizations/wolfgang-ch/rules?open=java%3AS116&rule_key=java%3AS116)) but you disabled the rule after [I found out](https://github.com/mytourbook/mytourbook/discussions/919#discussioncomment-3528036) that we couldn't specify a rule for public fields and a different rule for private fields